### DISCUSSION TITLE
toggle bookmark side bar

### DISCUSSION THREAD

**Discussion by MAFiA303** [Godiesc/firefox-gx#64] [Ideas]
great work! Your added features should be default in FF and I think it will become viral in the FF community soon

I have a small suggestion for the side bookmark
press `shift+ ctrl + B` to toggle bookmark
**result:**
only the icons disappear and not the sidebar
**expected result:**
for the sidebar to disappear

**Comment by Godiesc** [comment]
I recently updated the theme files for that extras functions jejej 
and the side bookmark, right know just work with "only show", it's difficul to me make a condition into the css code to make the function you want, but I hope in a near future do it if it is not imposible.

**Comment by Godiesc** [comment]
hi again,
after all this years your request was done, I don't know if you still use it but from Firefox v.134 you can toggle the sidebar with the shortcut without having misplaced elements.
